pkg/ddevapp/global_dotddev_assets/commands/web/npm

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
```

Comment:
```suggestion
#ddev-generated
```

This tells ddev that it's ok to replace it, because ddev created it. When you make one for your own use, you remove that, but we need it in there on one that ddev is responsible for. 

You can just click the commit button to commit this suggestion.